doc/static-of.md

```diff
@@ -0,0 +1,39 @@
+@property can-observable-array/array.static.of of
+@parent can-observable-array/static
+
+@description Static method to create an ObservableArray.
+
+@signature `ObservableArray.of(<TYPE>)`
+
+  Returns an `ObservableArray` constructor which can be used to define the [can-observable-object#Typedproperties Typed properties].
```

Comment:
I think this should first explain that it sets the [items](https://canjs.com/doc/can-observable-array/static.items.html) static property. Something like:

> Returns an `ObservableArray` constructor with its [static items](https://canjs.com/doc/can-observable-array/static.items.html) static property property set to the `Type` argument. This can be used to define the [can-observable-object#Typedproperties Typed properties].